README.md

```diff
@@ -54,7 +53,7 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 
 ## See also
 
-- [Developing Extensions in AL](../developer/devenv-dev-overview.md)
+- [Developing Extensions in AL](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-dev-overview)
```

Comment:
```suggestion
- [Developing Extensions in AL](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-dev-overview)
```
The locale part can be omitted, an automatic one will be picked based on the current location. For me, it picks `en-gb`.

Also, it's generally good practice to use forward links.